src/eduid_webapp/reset_password/app.py

```diff
@@ -56,10 +57,12 @@ def __init__(self, name: str, config: dict, **kwargs):
 
         # Register views
         from eduid_webapp.reset_password.views.reset_password import reset_password_views
-        self.register_blueprint(reset_password_views, url_prefix=self.config.application_root)
+        from eduid_webapp.reset_password.views.change_password import change_password_views
+        self.register_blueprint(change_password_views)
+        self.register_blueprint(reset_password_views)
 
         # Register view path that should not be authorized
-        self = no_authn_views(self, ['/reset-password.*'])
+        self = no_authn_views(self, [r'^(?!/chpass).*$'])
```

Comment:
no_authn_views already adds ^ and $ to each string in the list.

I would also keep it as simple as possible by listing only paths that should be whitelisted (unauthorized). Think of the input strings as paths or sub paths not regular expressions :)

Reply:
:+1:

Reply:
Amended here: https://github.com/SUNET/eduid-webapp/pull/288/commits/e6e48d9e479777ec2cb6ca3411ee2a591c4c7898

Also needs to merge PR #74 in eduid-developer